scripts/get_building_footprints.py

```diff
@@ -0,0 +1,55 @@
+"""
+Based on https://planetarycomputer.microsoft.com/dataset/ms-buildings#Example-Notebook
+The resulting GPKG is hosted on GCS natcap-urban-online-datasets
+"""
+
+import subprocess
+
+import dask_geopandas
+import deltalake
+import mercantile
+import pystac_client
+import planetary_computer
+import pygeoprocessing
+import shapely
+
+from osgeo import osr
+
+catalog = pystac_client.Client.open(
+    "https://planetarycomputer.microsoft.com/api/stac/v1",
+    modifier=planetary_computer.sign_inplace,
+)
+collection = catalog.get_collection("ms-buildings")
+
+srs = osr.SpatialReference()
+srs.ImportFromEPSG(4326)
+target_proj_wkt = srs.ExportToWkt()
+base_info = pygeoprocessing.get_raster_info('../appdata/lulc_overlay_3857.tif')
+bbox = pygeoprocessing.transform_bounding_box(
+    base_info['bounding_box'],
+    base_info['projection_wkt'],
+    target_proj_wkt
+)
+print(bbox)
```

Comment:
Just seeing if it's helpful to keep this print statement here.

Reply:
Yes, I think it is. This script is not run in production, it's meant to be run interactively.